packages/flutter/pubspec.yaml

```diff
@@ -19,5 +22,13 @@ dependencies:
   newton:
     path: ../newton
 
+dev_dependencies:
+  flx:
+    path: ../flx
+  flutter_test:
+    path: ../flutter_test
+  stocks:
+    path: ../../examples/stocks
```

Comment:
Why not move the stocks tests into the stocks app?  This dependency from production code to example code is backwards.